snmpy

```diff
@@ -32,8 +33,8 @@ def parse_conf(parser):
             background(conf['snmpy_global']['create_pid'])
 
         create_log(conf['snmpy_global']['logger_dest'])
-    except (IOError, yaml.parser.ParserError, yaml.scanner.ScannerError) as e:
-        parser.error('cannot parse configuration file: %s' % e)
+    except (IOError, yaml.YAMLError) as e:
+        raise ValueError('cannot parse configuration file: %s', e)
```

Comment:
This is a worse user experience.  parser.error will invoke `argparse`'s error which will print usage and provide a meaningful error message.  i.e.:

```
$ ./snmpy -f release.sh 
usage: snmpy [-h] [-f CONFIG_FILE] [-i INCLUDE_DIR] [-t PERSIST_DIR]
             [-r PARENT_ROOT] [-s SYSTEM_ROOT] [-l LOGGER_DEST]
             [-w HTTPD_PORT] [-p [PID_FILE]] [-m [MIB_FILE]] [-e KEY VAL] [-v]
snmpy: error: cannot parse configuration file: while scanning for the next token
found character '\t' that cannot start any token
  in "release.sh", line 5, column 1
```

With your changes it'll just throw an exception which is ugly.